apps/mull-ui/src/app/pages/create-event/create-event.tsx

```diff
@@ -21,6 +21,11 @@ export interface CreateEventProps {
   history: History;
```

Comment:
```suggestion
  history: IHistory;
```

and import the right History interface

```ts
import { History as IHistory } from 'history';
```

Reply:
No need for the alias for the interface btw, just  `import { History } from 'history';` works fine